src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -426,12 +427,32 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
 
         internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved)
         {
-            if (ReferenceEquals(_lazyDeclaredBaseType, ErrorTypeSymbol.UnknownResultType))
+            return GetDeclaredBaseType(basesBeingResolved, ignoreNullability: false);
+        }
+
+        private NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved, bool ignoreNullability)
```

Comment:
>basesBeingResolved [](start = 69, length = 18)

Why do we even have this parameter if it's never used?

Reply:
Good catch. I'll remove the parameter in a subsequent PR.

---
In reply to: [198643234](https://github.com/dotnet/roslyn/pull/28028#discussion_r198643234) [](ancestors = 198643234)